gulpfile.js

```diff
@@ -33,4 +33,17 @@ elixir(function(mix) {
         'socialmediashare.js'
     ], 'public/js/scripts.js', 'public/js');
 
+    /**
+     * Compile sass files to css
+     */
+    mix.sass('main.scss')
+       .sass('audioplayer.scss');
```

Comment:
Essentially, what sass does is compile to CSS.

**Running CSS from SASS**

When you run `gulp`,a CSS file will be **created or updated** from the sass file with the same name.

The upside to writing our CSS in SASS and not in the CSS files directly is that are able to check for syntax errors during compilation. 

While it is possible that I may forget to add a semi-colon at the end of a CSS property, running `gulp` highlight this error (_Line number and error extract_ ) out before it tries to minify. 

Like so.

<img width="1440" alt="screen shot 2016-03-31 at 14 44 12" src="https://cloud.githubusercontent.com/assets/16223627/14174741/252bcfac-f74f-11e5-9342-ac677dd580a1.png">

**Running CSS directly**

When working with CSS directly, an error in the syntax(_LIke the unnecessary dot on line 3 in the screenshot below)_, gulp will throw a none descriptive error.

<img width="1440" alt="screen shot 2016-03-31 at 14 47 56" src="https://cloud.githubusercontent.com/assets/16223627/14174841/c7bda4b6-f74f-11e5-8561-002cae3a5b4e.png">